src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,140 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+    private Node<T> temporaryNode;
+
+    private class Node<T> {
```

Comment:
All inner classes should be at the end of class

Reply:
not fixed